src/native/fs/fs_napi.cpp

```diff
@@ -77,7 +77,8 @@ struct FSWorker : public Napi::AsyncWorker
     std::string _backend;
     std::string _desc;
     int _errno;
-    int thresholdInMilliSec = 2;
+    protected:
+        int _warn_threshold_ms = 2;
```

Comment:
I think the warn threshold should generally be higher.
I would aim for 100 ms.
but in any case I think it should be set in config.js and passed to the fs workers in the config param (where we send uid/gid/backend).